app/src/main/java/net/skyscanner/backpack/demo/compose/DialogStory.kt

```diff
@@ -151,7 +151,26 @@ internal fun FlareDialogExample() =
                 painter = painterResource(R.drawable.canadian_rockies_canada),
                 contentDescription = stringResource(R.string.image_rockies_content_description),
                 contentScale = ContentScale.Crop,
-                modifier = Modifier.height(264.dp),
+                modifier = Modifier.height(192.dp),
+            )
+        }
+    }
+
+@Composable
+@DialogComponent
+@ComposeStory("Flare Vertical")
```

Comment:
you can set `kind = StoryKind.StoryOnly` here to not generate the screenshot

Reply:
it looks like it has been [changed to DemoOnly](https://github.com/Skyscanner/backpack-android/blob/9eda2a2a0ca504bc41c9ccbe590df1a463869ed3/app/src/main/java/net/skyscanner/backpack/demo/meta/Story.kt#L38) - I can update the CONTRIBUTING.md accordingly

Reply:
ah sorry about that - yes would be great if you could update it :)